src/components/Dao/DaoLandingPage.scss

```diff
@@ -15,7 +15,7 @@
   .headerText {
     font-size: 28px;
     font-weight: bold;
-    color: rgba(104, 155, 214, 1);
+    color: $title-color;
```

Comment:
how this related to 3box ?

Reply:
https://github.com/daostack/alchemy/pull/1947#issuecomment-658451101

Reply:
@roienatan asked to use color variables, a good request.